### DISCUSSION TITLE
Quorum for DCDAO Governance - Poll

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#70] [Polls]
DCDAO lets show how active we can be.

**Comment by 2xJDubs** [comment]
With DCDAO membership now at 15 people I would like to see us require at least 50% of members, 8, to reach quorum.  I think doing this will help increase participation from the community.

**Comment by Laki389** [comment]
I voted to keep it at 4 votes.

IMHO, important proposals, or controversial proposals, naturally attract more voters, lobbying to vote, and copy-paste proposals that are repeated and this quorum of 4 voters is now difficult to reach.

Here for example, this proposal has only 3 votes in total, the voting is over and if it goes according to the rules and regulations, the voting would be invalid.
https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xcbe5bce58db3cf1ba6de432603447631858545a2dc933130af3ffc02ebdc5c01

**Comment by hackfisher** [comment]
I should keep it at 4 votes. Changing to 50% will make us unable to pass any new proposals due to low participate.

And according to [constitution](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xd750140eccee3434f65feb156c985f6425ed884d56aa2a295c189d5e76fe9dc5), we should use number instead of percentage as quorum.

> Phase 2: The proposal should be accompanied by a Snapshot poll or other method as determined under the governance process, which can only be submitted by any membership. The Snapshot poll also runs for at least 1 week and is decided by a simple majority with minimum membership voting participating meeting the quorum requirement. The quorum is at least 4 but can be updated to a larger number by passing new non-constitution proposals to change it as more members join.

**Reply by SasoLithops** [reply]
I would disagree here. We differentiate from other DAOs in a way we know every single DCDAO member as each member needs to be approved by steering committee. If we are so careful at choosing DCDAO members why not teach them to actively participate in governance. We can always tag all when proposal is up. And 10 of us are Darwinia Devs, ambassadors and June. We need to be active by default. At least in my opinion as through governance you also learn about the progress of DCDAO.

Thank you for reminding. When proposal is up I will do in numbers and write percentage in brackets.

**Reply by hackfisher** [reply]
Teach is good, but the effect is not guaranteed, the only effective way for resolve low participation is config longer voting time.

Please see my answer below.

This DAO will be stall and useless if it cannot reach quorum, can not decide and losing vitality.

**Reply by hackfisher** [reply]
The proposals will be rejected if there are more against than for no matter how low the quorum is, long voting time is more effective than quorum, and non-voting members should be counted as abstain or don't care.

Quorum should be configured in a conservative strategy to ensure that the voting process can proceed

**Reply by SasoLithops** [reply]
I understand your concerns and your points presented they all make sense based on current voting participation. 
But my opinion is still that having a higher quorum and Abstain and No with veto option is way better.
If we keep it at four no one will vote as 4 Darwinia devs votes are enough and only "controversial decisions" will be more attended. No one will care about governance and participation in DCDAO will be low like it is now. 
We have different views which I feel is totally  fine. DCDAO community can decide together as we only 2 out of 14. 
This poll was just to see if there is any interest in change.
If quorum is 7(50% seems to be winning decision) after change and 10 of us "from Darwinia" won't that encourage us all to participate in each voting and understand that change. We can all tag each other when proposal live to go and vote.

**Comment by JuneCribb** [comment]
I picked 50% percent is a good idea, an average of the total members of DCDAO gets to vote. 

But what happens when we don't get a 50% vote on a proposal, will it pass or cause delay as Fisher said? 

To prevent stalling, I think 4/4 is a good number.

**Reply by hackfisher** [reply]
There are only two results: pass or rejected. I have zero affect to the voted results, I'm normal member just like you, no any special, constitution should be the fundamental rules.

We should config it according to the history not the assumption of the activity.

The quorum is not all of key factor as long as we have enough time for voting, just like the curve in the Polkadot open gov, the support aka. quorum could be low as 0.1%

<img width="1712" alt="image" src="https://github.com/dcdao/discussion/assets/1070122/a7f0f652-aa56-43c0-9e7e-4bd830c2dc3c">

We should keep use 4 for current participate, if more participates that's good we can decide quicker, if no more participates, let's the voters decide, otherwise, it will stall.

**Reply by hackfisher** [reply]
And once it stall, we may have to do an amendments to the constitution via a constitution proposal accordingly:

https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xd750140eccee3434f65feb156c985f6425ed884d56aa2a295c189d5e76fe9dc5

> Any change may be made to this constitution only by a two-thirds majority and at least 5 of all memberships participating.
> The process for making amendments should be
> Public: The amendment process should be open and transparent, allowing members and possibly the general public to be aware of the changes being proposed.
> 
> Notification to All Memberships: All members of the organization or body should be informed about the proposed amendments. This ensures that every member is allowed to understand and possibly contribute to the discussion regarding the changes.
> 
> Duration: The voting process should last at least 3 months. This duration ensures there is ample time for thorough discussion, debate, and consideration of the proposed amendments.

It takes at least 3 months, during that time the non-constitution proposals will be rejected if they do not reach the quorum.
